### PR TITLE
update dev setup into to replace npm by yarn

### PR DESCRIPTION
**Pull request by tcitworld**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,9 +66,9 @@ We’d like to thank [BrowserStack](https://www.browserstack.com) for providing
 
 ## Developer setup info
 
-Just clone this repo into your apps directory (Nextcloud server installation needed). Additionally,  [nodejs and npm](https://nodejs.org/en/download/package-manager/) are needed for installing JavaScript dependencies.
+Just clone this repo into your apps directory (Nextcloud server installation needed). Additionally,  [nodejs](https://nodejs.org/en/download/package-manager/) and [yarn](https://yarnpkg.com/) are needed for installing JavaScript dependencies.
```
Comment (georgehrke) [inline README.md]: Can you delete this change please :)

**Comment by skjnldsv** [comment]
Well I guess not! 😆 😁

**Comment by georgehrke** [comment]
Oh, sorry ^^

**Comment by georgehrke** [comment]
I will check and send a pr

**Comment by tcitworld** [comment]
@georgehrke Also, you're sure we require node6 ?

**Comment by georgehrke** [comment]
Yes, I think so.
I recently had someone in IRC who tried node 0.8.something and everything was broken for him.
Also Node 6.x are 7.x are the only supported node versions. Everything older is EOL

**Comment by tcitworld** [comment]
Indeed. Node 0.x is definitely to bad. Debian sid still gives v4.x, that's why I asked.

**Comment by georgehrke** [comment]
What? oO 

Debian Sid provides v4? WTF
Maybe its so old that its unstable again ...

I never tested node 4.
Only 6 and 7.

**Comment by tcitworld** [comment]
Yeah, I have 6 install, but through the nodesource repo. Debian maintainers can't keep up I guess.

**Comment by georgehrke** [comment]
Hm, then they should just remove it :/

**Pull request merged**